src/main/java/org/betonquest/betonquest/modules/web/updater/UpdateSourceHandler.java

```diff
@@ -93,9 +93,9 @@ Pair<Version, String> searchUpdate(final UpdaterConfig config, final Version cur
     }
 
     /**
-     * A consumer for {@link UpdateSource}s.
+     * A consumer for any {@link DevelopmentUpdateSource} or {@link ReleaseUpdateSource}.
      *
-     * @param <T> something implementing {@link UpdateSource}
+     * @param <T> something that is a {@link DevelopmentUpdateSource} or {@link ReleaseUpdateSource}
      */
     private interface UpdateSourceConsumer<T> {
```

Comment:
Both the old and new documentation is a lie: There is no constraint on the generic type `T`, so this interface could consume anything!